pkg/controller/util/node/controller_utils.go

```diff
@@ -463,3 +464,16 @@ func ValidateUrl(urlString string) error {
 
 	return nil
 }
+
+func GetTenantPartitionClientsFromFiles(kubeconfigFiles []string) ([]clientset.Interface, error) {
```

Comment:
Perhaps rename to GetClientsFromKubeConfigs and move to shared util

Reply:
this is intentionally kept here. the shared package only provides the functions to get the kubeconfig,  for array of the kubeconfig etc, it belongs to the caller's logic.